pom.xml

```diff
@@ -170,6 +169,19 @@
                 <spaces>true</spaces>
               </indent>
             </format>
+            <format>
+              <includes>
+                <include>**/*.md</include>
+              </includes>
+              <excludes>
+                <exclude>**/target/**</exclude>
+              </excludes>
+              <prettier>
```

Comment:
For posterity, can you please add a comment why you chose to use this formatter?

Reply:
done